### DISCUSSION TITLE
Overriding plugin functionality

### DISCUSSION THREAD

**Discussion by bockstaller** [pretix/pretix#2106] [Q&A]
I need to implement an organization specific exporter for our sepa direct debit mandates (CSV for our ERP). Preferably by repurposing the existing menu entry provided by the plugin. 
I could create a new plugin, which disconnects the sepadebits-exporter views and provides my own, or fork the plugin and make the changes directly.
Do you have any preexisting experience with this situation?

**Comment by raphaelm** [comment]
Hm, you'd probably need some serious monkey patching. Should be possible, but sounds doomed to get really hacky and confusing if the plugin changes. It might might be to just run a custom branch of the plugin.
